src/coreclr/tests/src/Directory.Build.props

```diff
@@ -139,6 +139,7 @@
   <PropertyGroup>
     <TargetsUnknownUnix Condition="'$(TargetsUnix)' == 'true' AND '$(TargetOS)' != 'FreeBSD' AND '$(TargetOS)' != 'Linux' AND '$(TargetOS)' != 'NetBSD' AND '$(TargetOS)' != 'OSX' AND '$(TargetOS)' != 'illumos' AND '$(TargetOS)' != 'Solaris'">true</TargetsUnknownUnix>
     <Language Condition="'$(Language)' == '' and  '$(MSBuildProjectExtension)' == '.csproj'">C#</Language>
+    <Language Condition="'$(Language)' == '' and  '$(MSBuildProjectExtension)' == '.fsproj'">F#</Language>
```

Comment:
Should this be shared across all subsets? https://github.com/dotnet/runtime/pull/39828#pullrequestreview-454505947 is adding an F# project as well.

Reply:
It's a good question. I have a feeling it is probably something that should eb done with larger refactors. However, I am open to suggestions.

Reply:
Maybe all the `Language` properties can be shared. Probably out of the scope of this PR since it is just following what we already have. 

cc: @ViktorHofer

Reply:
Shouldn't that be already set by the sdk?

Reply:
The SDK provided property seems work in `.targets`, but not `.props`.

Reply:
Why do you need it in the props file? This isn't intended to be set before the project is evaluated.

Reply:
From a quick GH search it seems like we don't use Language at any props or targets level so this is just used by the SDK.

Reply:
Which means we can remove that property here?

Reply:
I would prefer to leave this to its own PR.